winml/lib/Api.Core/ModelInfo.cpp

```diff
@@ -137,13 +137,22 @@ std::unique_ptr<onnx::ModelProto>
 WinML::CreateModelProto(
     const char* path) {
   int file_descriptor;
+  _set_errno(0); // clear errno
   _sopen_s(
```

Comment:
Why wouldn't check the returned value of '_sopen_s' instead? So that you don't need to clean errno every time.